wix/Resources/license.rtf

```diff
@@ -0,0 +1,92 @@
+{\rtf1\ansi\ansicpg1252\deff0\deflang1031{\fonttbl{\f0\fnil\fcharset0 MS Shell Dlg 2;}{\f1\fnil\fcharset0 Calibri;}}
+{\*\generator Msftedit 5.41.21.2510;}\viewkind4\uc1\pard\f0\fs17 GNU GENERAL PUBLIC LICENSE\par
+Version 2, June 1991 \par
+\par
+Copyright (C) 1989, 1991 Free Software Foundation, Inc.  \par
+59 Temple Place - Suite 330, Boston, MA  02111-1307, USA\par
+\par
+Everyone is permitted to copy and distribute verbatim copies\par
+of this license document, but changing it is not allowed.\par
+\par
+Preamble\par
+The licenses for most software are designed to take away your freedom to share and change it. By contrast, the GNU General Public License is intended to guarantee your freedom to share and change free software--to make sure the software is free for all its users. This General Public License applies to most of the Free Software Foundation's software and to any other program whose authors commit to using it. (Some other Free Software Foundation software is covered by the GNU Library General Public License instead.) You can apply it to your programs, too. \par

```

Comment:
I'm not sure this is the GPLv2+CPE license.  It looks like it's just the GPLv2?  Also need to check with the Eclipse OpenJ9 folks as to what license would be deployed for them.

Reply:
Oh, thats correct. I started with the license included in the hotspot package and missed to review the other package about this. Just thought it is the same.